#goal definition
float64 base
float64 shoulder
float64 elbow
float64 wrist1
float64 wrist2
float64 wrist3
string head
---
#result definition
float64[] current
---
#feedback
float64[] final
